examples/benchmarks/README.md

```diff
@@ -134,7 +134,7 @@ If you want to contribute your new models, you can follow the steps below.
     - `README.md`: a brief introduction to your models
     - `workflow_config_<model name>_<dataset>.yaml`: a configuration which can read by `qrun`. You are encouraged to run your model in all datasets.
 3. You can integrate your model as a module [in this folder](https://github.com/microsoft/qlib/tree/main/qlib/contrib/model).
-4. Please updated your results in the benchmark tables, e.g. [Alpha360](#alpha158-dataset), [Alpha158](#alpha158-dataset)(the values of each metric are the mean and std calculated based on 20 runs with different random seeds, if you don't have enough computational resource, you can ask for help in the PR).
+4. Please update your results in the above **Benchmark Tables**, e.g. [Alpha360](#alpha158-dataset), [Alpha158](#alpha158-dataset)(the values of each metric are the mean and std calculated based on **20 Runs** with different random seeds. You can accomplish the above operations through the automated [script](https://github.com/microsoft/qlib/blob/370477288dcf8fb2abbb7e7bba69cf6f841d959b/examples/run_all_model.py#L286) provided by Qlib, and get the final result in the .md file. if you don't have enough computational resource, you can ask for help in the PR).
```

Comment:
The link should not be bound to a specific version.
https://github.com/microsoft/qlib/blob/370477288dcf8fb2abbb7e7bba69cf6f841d959b/examples/run_all_model.py#L286